tsconfig.json

```diff
@@ -1,16 +1,12 @@
 {
   "compilerOptions": {
     "lib": ["es2022"],
-    "moduleResolution": "NodeNext",
-    "module": "NodeNext",
+    "moduleResolution": "Bundler",
+    "module": "es2022",
     "allowJs": true,
     "checkJs": true,
     "noEmit": true,
-    "declaration": true,
-    "declarationMap": true,
```

Comment:
It does. One could remove `tsconfig.json` entirely though it can be handy to have a minimal `tsconfig.json` or `jsconfig.json` that has `moduleResolution: "bundler"` and `"types": ["vitest/globals"]` for IDE purposes.

Previously `tsconfig.json` was used for type declaration purposes and it can be shortened even more now, but I left the majority of things intact because I don't know your IDE / dev environment.